backend/src/ml_space_lambda/authorizer/lambda_function.py

```diff
@@ -324,6 +326,15 @@ def lambda_handler(event, context):
                                 policy_statement["Effect"] = "Allow"
                     elif target_type == DatasetType.PRIVATE and username == target_scope:
                         policy_statement["Effect"] = "Allow"
+                    elif target_type == DatasetType.GROUP:
+                        if Permission.ADMIN in user.permissions:
+                            policy_statement["Effect"] = "Allow"
+                        else:
+                            # TODO: check if this user is a member of the Group
```

Comment:
Is this still a TODO? Looks like you did the damn thing.

Reply:
I *did* do the damn thing didn't I?